### ISSUE TITLE
Can the `expect` API be used in a non-test environment?

### ISSUE THREAD

**Issue by yjhmelody** [jtenner/as-pect#350]
Can the `expect` API be used in a non-test environment? Because wasm has many different environments (with various host functions), sometimes the unit test environment cannot test some wasm environments, but these APIs of `as-pect` are very useful for testing. Can they be used as ordinary Library introduction?

**Comment by jtenner** [comment]
No. They are coupled to the host at the moment. The reason why is because the Reflect api has to tell the host the shape of the expected and actual values before performing assertions. However, it's possible to create a version that simply performs a set of assertions. I can help with this :)

**Comment by jtenner** [comment]
If this is all set, feel free to close this issue.

**Comment by jtenner** [comment]
@yjhmelody do you have experience with transforms?
